src/InputWithLabel.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useRef } from 'react';
+
+const InputWithLabel = (props) => {
+    const inputRef = useRef(null);
+
+    useEffect(() => {
+      inputRef.current.focus();
+    }, []);
+
+    return (
+    <>
+    <label htmlFor={props.id}>{props.children}</label>
```

Comment:
Add one more indent level to this `label` element because it is inside the fragment.
You can actually use VSCode shortcut `Alt + Shift + F` to format the whole file.